src/data-grid/README.md

```diff
@@ -1,34 +1,50 @@
 # Visual Studio Code Data Grid
 
-The `vscode-data-grid` enables developers to display data in a tabular layout. The data grid is created using three components that work together:
+The `vscode-data-grid` enables developers to display data in a tabular layout.
+
+![Data grid hero](/docs/assets/data-grid-hero.png)
+
+The data grid is created using three components that work together:
 
 -   `<vscode-data-grid>`: The top level container element.
 -   `<vscode-data-grid-row>`: Displays a single row of data associated with a single record or a header row.
 -   `<vscode-data-grid-cell>`: Displays a single cell of data within a row.
 
-## Data Grid Attributes
+## Usage
+
+| ❌ Don't                                               | ✅ Do                                                           |
+| ------------------------------------------------------ | --------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)          |
+| Don't use a data grid in place of a basic list.        | Use data grids to list complex data with secondary information. |
+
+| ❌ Don't                                                             | ✅ Do                                                                                           |
+| -------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png)               | ![Image placeholder](/docs/assets/img-placeholder.png)                                          |
+| Don't use data grids in small containers unless absolutely necessary | Place data grids in a main content area to display as much information in one view as possible. |
```

Comment:
Could we rephrase "Place data grids in a main content area to display as much information in one view as possible." to "Place data grids in a main content area to display as much information as possible in one view."?

I think the latter reads a bit better.